### PR TITLE
LPS-148033 form filed project should to update node_modules

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
@lawrence-lee I checked the project templates test. Its react npm portlet test failure. Seiphon is doing it now. @Seiphon,

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-form-field/src/main/java/com/liferay/project/templates/form/field/internal/FormFieldProjectTemplateCustomizer.java [hunk 1]
```diff
@@ -122,7 +121,9 @@ public void onAfterGenerateProject(
 				String nodeManager = gradleProperties.getProperty(
 					"liferay.workspace.node.package.manager");
 
-				if (!StringUtils.equals(nodeManager, "npm")) {
+				if (Validator.isNull(nodeManager) ||
```
Comment (lawrence-lee) [inline modules/sdk/project-templates/project-templates-form-field/src/main/java/com/liferay/project/templates/form/field/internal/FormFieldProjectTemplateCustomizer.java:124]: I think we should solve this the other way. Treat yarn as default and then make the change if the user has npm. Then we can add a test case for this to check to make sure this customizer code is working.

**Pull request closed without merging**